### ISSUE TITLE
0.222: asmjit fails to build on aarch64

### ISSUE THREAD

**Issue by belegdol** [mamedev/mame#6861]
0.222 fails to build on aarch64 due to asmjit. 3rdparty/asmjit/src/asmjit/core/arch.cpp contains the following section
```c++
#ifdef ASMJIT_BUILD_ARM
  #include "../arm/armarchdata_p.h"
#endif
```
But there is no arm folder in 3rdparty/asmjit/src/asmjit, only core and x86.
Looks like a conditional is needed to only attempt to build asmjit on x86.

**Comment by Tafoid** [comment]
Was also reported here:
https://mametesters.org/view.php?id=7687

**Comment by belegdol** [comment]
Resolved by #6862
